src/screens/CreatePost/elements/Card.js

```diff
@@ -1,51 +1,52 @@
 import React from 'react';
-import {View, StyleSheet, Platform} from 'react-native';
-import {COLORS} from '../../../utils/theme';
-
-const Card = (props) => {
+import { View, StyleSheet, Platform } from 'react-native';
+import { colors } from '../../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use a different color scheme. Ensure that the new color scheme (`colors`) is correctly defined and contains all the necessary colors.

```diff
- import {COLORS} from '../../../utils/theme';
+ import {colors} from '../../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->